dss-core/src/test/java/io/github/ztkmkoo/dss/core/actor/AbstractDssActorTest.java

```diff
@@ -19,12 +20,12 @@
 
     protected AbstractDssActorTest() {}
 
-    @BeforeClass
+    @BeforeAll
```

Comment:
I am not sure for those `nullpointerexception` is related with surefire version.
Because in my local, `nullpointerexception` didn't appear. 

[test-build-maven-engines](https://junit.org/junit5/docs/current/user-guide/#running-tests-build-maven-engines-configure) In this document, it describe surefire give "native support for executing tests on the JUnit Platform", like real travis CI environment. (IntelliJ supports Junit test runner)

But I have to study more to explain exact dignose.